project/profiles/capacity4more/modules/c4m/content/c4m_content_group/c4m_content_group.block.inc

```diff
@@ -155,9 +155,9 @@ function _c4m_content_group_block_header_name_banner() {
   elseif (isset($wrapper->group_access)) {
     // Determine access per group_access parameter.
     $group_access = $wrapper->group_access->value() ? 'private' : 'public';
-    $variables['group_access'] = $group_access;
   }
 
+  $variables['group_access'] = $group_access;
```

Comment:
@ybaras 
Let's just use `c4m_og_get_access_type()` in here and remove all the unnecessary logic from this function ...